contents/tutorials/build-site-app.md

```diff
@@ -0,0 +1,191 @@
+---
+title: How to build a site app
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-10-27
+featuredImage: ../images/tutorials/banners/build-site-app.png
+topics: ["apps"]
+---
+
+Site apps make it quick and easy to add features like forms and banners to your site through our JavaScript library. This enables you to do things like capture feedback, add notifications, provide support, and more. These apps can then capture data for analysis in PostHog.
+
+In this tutorial, we’ll teach you how to build a site app. To complete this tutorial, you only need [Node installed](https://nodejs.org/en/download/) to build and run the app.
```

Comment:
I think it's worth adding here what it is that the site will do. We should also, at some point in the intro, point to the docs so users can get more info on site apps (and apps in general).